src/manageiq_client/api.py

```diff
@@ -115,6 +115,16 @@ def delete(self, url, **payload):
                 raise APIException("JSONDecodeError: {}".format(data.text))
         return self._result_processor(data)
 
+    def options(self, url, **opt_params):
+        self.logger.info("[RESTAPI] OPTIONS %s %r", url, opt_params)
+        data = self._sending_request(
+            partial(self._session.options, url, params=opt_params, verify=False))
```

Comment:
I forgot to change this, we don't have support for ssl verification in out bundled client. I'll fix this.